client/layout/index.jsx

```diff
@@ -121,5 +123,11 @@ module.exports = React.createClass( {
 					isActive={ translator.isActivated() }/>
 			</div>
 		);
+
+		return (
+			<Provider store={ this.props.store }>
```

Comment:
You will have problems with everything that uses `context.layout.setState`, because it will set the state of `Provider`, not `Layout`